### PR TITLE
[1LP][RFR]Added test_refresh_with_stack_without_parameters

### PR DESCRIPTION
**Pull request by mmojzis**
Added automated test for refresh_with_stack_without_parameters
{{ pytest: -k "test_refresh_with_stack_without_parameters" -v }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 60086949b4ca7b9a9aaf002eaeeb011c64f6ec87

The *local* fixture **`stack_without_parameters`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**
    - *test_refresh_with_stack_without_parameters*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 61ad4f05c5b6602b823ca239b63c7eb5a1ef779b

The *local* fixture **`stack_without_parameters`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**
    - *test_refresh_with_stack_without_parameters*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 730eca79e2ced66bfb287baf4720d5d52ec25680

The *local* fixture **`stack_without_parameters`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**
    - *test_refresh_with_stack_without_parameters*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 137bba8261056a524cce6d46b056d23bcbcc2ae0

The *local* fixture **`stack_without_parameters`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**
    - *test_refresh_with_stack_without_parameters*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/cloud/test_providers.py [hunk 1]
```diff
@@ -198,6 +198,18 @@ def instance_with_ssh_addition_template(appliance, provider):
     instance.delete()
 
 
+@pytest.fixture
+def stack_without_parameters(provider):
+    stack = provider.mgmt.create_stack(name=fauxfactory.gen_alpha(10),
+        template_url="https://s3-us-west-2.amazonaws.com/cloudformation-templates-us-west-2/Managed"
+                     "_EC2_Batch_Environment.template",
```
Comment (john-dupuy) [inline cfme/tests/cloud/test_providers.py]: Q: Can we pull this template from yamls?

Comment (mmojzis) [reply cfme/tests/cloud/test_providers.py]: Well, sure. There is no problem, but this template url won't change. I am manually testing this for three years and url is still the same.

Comment (john-dupuy) [reply cfme/tests/cloud/test_providers.py]: While that is true, I think it is good to separate data from code.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
I detected some fixture changes in commit f82b595f191705bd278eb42a1aa9f20ae8d64b52

The *local* fixture **`stack_without_parameters`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**
    - *test_refresh_with_stack_without_parameters*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit e157a28ac21ac1ca7293111634b7bc7440e36974

The *local* fixture **`stack_without_parameters`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**
    - *test_refresh_with_stack_without_parameters*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by mmojzis** [comment]
@john-dupuy Yes, it passed now. I thought it was already merged so that's why I set it RFR.

File: cfme/tests/cloud/test_providers.py [hunk 2]
```diff
@@ -1169,8 +1180,9 @@ def test_create_azure_vm_from_azure_image(connect_az_account, cfme_vhd, upload_i
 
 
 @test_requirements.ec2
-@pytest.mark.manual
-def test_ec2_refresh_with_stack_without_parameters():
+@pytest.mark.usefixtures('has_no_cloud_providers')
+@pytest.mark.provider([EC2Provider], scope="function", override=True)
```
[the remark this answers is not in the document]
Comment (mmojzis) [reply cfme/tests/cloud/test_providers.py]: I already removed that, but it was not updated in PR: https://github.com/mmojzis/integration_tests/commit/d2d4ab0ffbbf0d0fc9bf7da1bc56ed744b72ca4e

**Pull request merged**